doc/kuksa_analysis/kuksa_requirementss.md

```diff
@@ -0,0 +1,202 @@
+
+### FR2-AD
+    Title: Support for VSS Metadata Elements by Databroker
+
+    Status: 🟡
+
+    Description: The Databroker shall support at least those metadata elements as defined by the VSS rule set.
+
+### FR3-AD
+    Title: Subscription Management by Databroker
+
+    Status: 🟡
+
+    Description: The Databroker shall keep a local record of all subscriptions of signal consumers.
+      - Detail:
+          The Databroker shall add or remove subscriptions to a subscription pool according to the subscription requests of the Signal Consumer.
+
+### FR4-AD
+    Title: Command Transmission Capabilities of Databroker to Provider
+
+    Status: 🟡
+
+    Description: The Databroker shall be able to send to the Provider the following commands
+    * Actuate on actuator.
+    * Start receiving signal values from the Provider.
+    * Stop receiving signal values from the Provider.
+
+## As Provider
+### FR1-AP
+    Title: Provider Claim Management by Databroker
+
+    Status: 🟡
+
+    Description: The data broker shall offer a method to clients allowing to claim providership of a set of signals.
+      a. IF all claimed signals are known AND
+            the client has providing rights for all claimed signals AND
+            all claimed signal is NOT yet claimed by another provider
+          THEN the data broker shall accept and remember the claim
+          ELSE the data broker shall reject the overall claim and return an error containing the reason.
+      b. The data broker shall remember accepted claims of a provider if the connection to the provider is lost.
+      c. The data broker shall allow providers to re-claim previously successfully claimed signals.
+   
+### FR2-AP
+    Title: Subscription and Actuation Notification Handling for Providers by Databroker
+
+    Status: 🟡
+
+    Description: The data broker shall offer a method to providers to subscribe for actuation notifications on a (sub-) set of claimed signals.
+      a. The data broker shall reject subscription requests containing signals where the client is not the provider of.
+      b. The data broker shall notify the provider of a signal about received actuation requests.
+
+### FR3-AP
+    Title: Signal State Update Mechanism for Providers by Databroker
+
+    Status: 🟡
+
+    Description: The data broker shall offer a method to providers to update the current state of a set of signals.
+      a. The current state consists a timestamp and either of valid value or a failure state.
+      b. The data broker shall offer a method optimized for frequent updates.
+      c. The data broker should offer a second method for non-frequent updates that is easy to use in a provider's implementation.
+      d. The data broker shall reject updating the current state of signals where the client is not the provider of.
+      e. The data broker shall store the updated value or failure state of a signal.
+
```

Comment:
We could add here one more point regarding VISS. How much we will support it. If we want to support it etc. Maybe also stating the problems we see with VISS